conans/client/conan_command_output.py

```diff
@@ -71,7 +71,7 @@ def _read_dates(self, deps_graph):
         for node in sorted(deps_graph.nodes):
             ref = node.ref
             if node.recipe not in (RECIPE_CONSUMER, RECIPE_VIRTUAL, RECIPE_EDITABLE):
-                manifest = self._cache.package_layout(ref).recipe_manifest()
+                manifest = self._cache.get_ref_layout(ref).recipe_manifest()
```

Comment:
@czoido @memsharded should I have used `get_latest_` instead?

Reply:
When the graph is resolved, the ``ref`` should already have the ``revision``. Internally, it has an assert for it:

```python
 def get_reference_layout(self, ref: ConanReference):
        assert ref.rrev, "Recipe revision must be known to get the reference layout"
```